ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -213,6 +213,9 @@ sleep 10m
 
 check_clusteroperators_status
 
+# Make sure no tests interfere with $KUBECONFIG
+sudo chattr +i $KUBECONFIG
```

Comment:
Looks like the container running the tests will need CAP_LINUX_IMMUTABLE  capability added